bin/oref0-bluetoothup.sh

```diff
@@ -63,12 +75,13 @@ if ( hciconfig -a hci${adapter} | grep -q "DOWN" ) ; then
  fi
 
 if !( hciconfig -a hci${adapter} | grep -q $HOSTNAME ) ; then
-      # Not sure that this is needed on Stretch, add an is_debian_jessie check here if something different required.
-      echo Bluetooth hci name does not match hostname: $HOSTNAME. Setting bluetooth hci name.
-      sudo hciconfig hci${adapter} name $HOSTNAME
-      if !( hciconfig -a hci${adapter} | grep -q $HOSTNAME ) ; then
-         hciconfig -a hci${adapter}
-         echo "$(date) Failed to set bluetooth hci name. Stop bluetoothd and allow next cycle to handle restart."
-         stop_bluetooth
+       # Not sure that this is needed on Stretch, add an is_debian_jessie check here if something different required.
+       echo Bluetooth hci name does not match hostname: $HOSTNAME. Setting bluetooth hci name.
+       sudo hciconfig hci${adapter} name $HOSTNAME
+       wait_for_hci_name
+       if ! ( hciconfig -a hci${adapter} | grep -q "$HOSTNAME" ) ; then
```

Comment:
This indentation is now inconsistent with the `fi` below. Was this whitespace change intentional?

Reply:
Good catch.  Had ignore whitespace turned on by accident in my diff tool.  However, the original indentation was not consistent with the rest of the file.  So latest commit makes it all consistent.